src/code/z_message.c

```diff
@@ -20,8 +41,8 @@ s32 D_801D02F8[15] = { 0,1,2,3,4,0,1,0,0,0,0,0,1,1,0 };
 
 #endif
 
-extern u16 D_801C6AB8[];
-extern u16 D_801C6B28[];
+extern u16 sBombersNotebookEventMessages[];
+extern u16 gBombersNotebookWeekEventFlags[];
```

Comment:
These can be externed in z64message.h (and also removed from variables.h, and given `BOMBERS_NOTEBOOK_EVENT_MAX` as the previous comment). Don't worry about the other message variables though if you don't want, just these two